resources/qml/pages/ContactsPage.qml

```diff
@@ -77,22 +77,22 @@ Page {
                     }
                     Label {
                         id: jidId;
-                        text: jid;
+                        text: qsTr("jid");
                         color: Theme.secondaryColor;
                         font.pixelSize: Theme.fontSizeTiny;
                     }
                 }
                 Label {
                     id: statusId;
-                    text: status;
+                    text: qsTr("status");
```

Comment:
This is the status which gets replaced by c++ code with the contact status. Not translateable.